FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_Echo_Qemu_mps2/main_networking.c

```diff
@@ -145,12 +157,30 @@ void main_tcp_echo_client_tasks( void )
      * vApplicationIPNetworkEventHook() below).  The address values passed in here
      * are used if ipconfigUSE_DHCP is set to 0, or if ipconfigUSE_DHCP is set to 1
      * but a DHCP server cannot be contacted. */
-    FreeRTOS_debug_printf( ( "FreeRTOS_IPInit\n" ) );
-    FreeRTOS_IPInit( ucIPAddress,
-                     ucNetMask,
-                     ucGatewayAddress,
-                     ucDNSServerAddress,
-                     ucMACAddress );
+    memcpy(ipLOCAL_MAC_ADDRESS, ucMACAddress, sizeof ucMACAddress);
```

Comment:
Is this memcpy common to both old and new

Reply:
Yes , this memcpy is common to both old and new, since with EndPoint changes, we call **FreeRTOS_FillEndPoint**which requires a initial MAC address from the ipLOCAL_MAC_ADDRESS.